client/jetpack-cloud/sections/partner-portal/primary/product-price-with-discount-info/index.tsx

```diff
@@ -26,6 +27,8 @@ export default function ProductPriceWithDiscount( {
 	const userProducts = useSelector( ( state ) => getProductsList( state ) );
 	const isDailyPricing = product.price_interval === 'day';
 
+	const isBundle = isEnabled( 'jetpack/bundle-licensing' ) && quantity > 1;
```

Comment:
I'm not certain whether it's necessary to check for `isEnabled('jetpack/bundle-licensing')` here. Do all consuming components only supply the quantity when we are loading from the new issue license page?

Reply:
I have kept this to be on the safer side since this component is being used in a lot of places.

Reply:
Would it make sense to put this responsibility on the consuming component? I believe we don't have any production UI that renders a bundle license, so I think it is safe.

I'll make this comment a non-blocker and approve the PR.